api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -415,6 +417,17 @@ message Cluster {
     bool use_http_header = 1;
   }
 
+  message KeyLbConfig {
+    // The key into the metadata.
+    string key = 1 [(validate.rules).string = {min_bytes: 1}];
+
+    LbPolicy fallback_policy = 2 [(validate.rules).enum = {defined_only: true in: 0 in: 1 in: 3}];
+
+    oneof fallback_lb_config {
+      LeastRequestLbConfig least_request_lb_config = 3;
```

Comment:
Why least request?

Reply:
Of the supported fallbacks in the enum validation on `fallback_policy`, least-req is the only one that takes additional config.